kong/plugins/session/session.lua

```diff
@@ -9,66 +9,67 @@ local ipairs = ipairs
 local _M = {}
 
 
-local function get_opts(conf)
-  local opts = {
-    name    = conf.cookie_name,
-    secret  = conf.secret,
-    storage  = conf.storage,
-    cookie  = {
-      lifetime   = conf.cookie_lifetime,
-      idletime   = conf.cookie_idletime,
-      path       = conf.cookie_path,
-      domain     = conf.cookie_domain,
-      samesite   = conf.cookie_samesite,
-      httponly   = conf.cookie_httponly,
-      secure     = conf.cookie_secure,
-      renew      = conf.cookie_renew,
-      discard    = conf.cookie_discard,
-      persistent = conf.cookie_persistent,
-    }
-  }
-
-  if conf.storage == "kong" then
-    opts.strategy = 'regenerate'
-    opts.storage = kong_storage
-  end
-
-  return opts
-end
-
-
 --- Open a session based on plugin config
 -- @returns resty.session session object
 function _M.open_session(conf)
-  return resty_session.open(get_opts(conf))
+  kong.log.inspect(conf.response_headers)
+
+  return resty_session.open({
+    secret                    = conf.secret,
+    audience                  = conf.audience,
+    storage                   = conf.storage == "kong" and kong_storage,
```

Comment:
just checking: storage being false loads cookie storage, right?

Reply:
yes `nil` and `"cookie"` are synonyms.